draft-ietf-quic-transport.md

```diff
@@ -4276,12 +4280,12 @@ An endpoint uses a STOP_SENDING frame (type=0x05) to communicate that incoming
 data is being discarded on receipt at application request.  This signals a peer
 to abruptly terminate transmission on a stream.
 
-Receipt of a STOP_SENDING frame is only valid for a send stream that exists and
-is not in the "Ready" state (see {{stream-send-states}}).  Receiving a
-STOP_SENDING frame for a send stream that is "Ready" or non-existent MUST be
-treated as a connection error of type PROTOCOL_VIOLATION.  An endpoint that
-receives a STOP_SENDING frame for a receive-only stream MUST terminate the
-connection with error PROTOCOL_VIOLATION.
+Receipt of a STOP_SENDING frame is invalid for a locally-initiated stream which
+does not exist or is in the "Ready" state (see {{stream-send-states}}).
```

Comment:
"does not exist" sounds a bit weird. In my implementation, when a stream is fully closed, I delete it from my map of streams at some point. At that moment, it does not exist any more, but receipt of a (delayed) STOP_SENDING is still a valid thing.

Maybe we can say "has not yet been created".

Reply:
```suggestion
has not yet been created or is in the "Ready" state (see {{stream-send-states}}).
```